### ISSUE TITLE
README.md can be improved

### ISSUE THREAD

**Issue by danielfcastro** [TA-Lib/ta-lib-python#272]
At line 273 until 281 the old content

```python
import talib

# list of functions
print talib.get_functions()

# dict of functions by group
print talib.get_function_groups()
```
Could be better written if we use the content below.

```python
import talib

# list of functions
print('Functions')
print("\n".join(talib.get_functions()))

# dict of functions by group
print('Functions Group')
print('\r\n')
print("\n".join(talib.get_function_groups()))
```

**Comment by mrjbq7** [comment]
Yours might be better for printing out, but I intended to just demonstrate those functions existed... maybe something like this?

```python
for name in talib.get_functions():
    print(name)

for group, names in talib.get_function_groups():
    print(group)
    for name in names:
        print(name)
```

**Comment by danielfcastro** [comment]
Yes, it is also another way.
With a needles complexity declaring the 2 for loops, but it is a way.


[image: photo]
*Daniel Ferreira Castro*

+5521988920009 | dfcastro@gmail.com

Skype: danielfcastro <#>
<http://www.facebook.com/daniel.ferreira.castro>
<http://twitter.com/danielfcastro>
<http://br.linkedin.com/in/danielfcastro/>
<https://www.linkedin.com/in/danielfcastro/>
Get your own email signature
<https://www.wisestamp.com/signature-in-email?utm_source=promotion&utm_medium=signature&utm_campaign=get_your_own>

Em qua, 24 de jul de 2019 às 15:15, John Benediktsson <
notifications@github.com> escreveu:

**Comment by mrjbq7** [comment]
That's fair, but just doing ``'\n'.join(stuff)`` doesn't really demonstrate what those functions return either...

**Comment by danielfcastro** [comment]
That is not the point.  The idea was just to suggest a better approach for
a kind of test file in python that lists the functions in a more textual
way.


[image: photo]
*Daniel Ferreira Castro*

+5521988920009 | dfcastro@gmail.com

Skype: danielfcastro <#>
<http://www.facebook.com/daniel.ferreira.castro>
<http://twitter.com/danielfcastro>
<http://br.linkedin.com/in/danielfcastro/>
<https://www.linkedin.com/in/danielfcastro/>
Get your own email signature
<https://www.wisestamp.com/signature-in-email?utm_source=promotion&utm_medium=signature&utm_campaign=get_your_own>

Em qui, 25 de jul de 2019 às 01:01, John Benediktsson <
notifications@github.com> escreveu:

**Comment by mrjbq7** [comment]
Fixed in b2c332e3b57ab33cc1bebc131033c8e618100111
